extension/scripts/content/global/ttGlobal.entry.js

```diff
@@ -1,70 +1,234 @@
 "use strict";
 
+let featureManager, globalFeatures;
+
 (async () => {
 	await loadDatabase();
 	console.log("TT: Global Entry - Loading script. ");
 
-	storageListeners.settings.push(loadGlobalEntry);
-	storageListeners.userdata.push(loadGlobalEntry);
-
-	await loadGlobalEntry();
-
-	console.log("TT: Global Entry - Script loaded.");
-})();
+	featureManager = new FeatureManager();
+	globalFeatures = new GlobalFeatures();
 
-async function loadGlobalEntry() {
-	if (settings.pages.global.alignLeft) document.documentElement.classList.add("tt-align-left");
-	else document.documentElement.classList.remove("tt-align-left");
+	globalFeatures.init();
 
-	document.documentElement.style.setProperty("--torntools-hide-upgrade-button", settings.pages.global.hideLevelUpgrade ? "none" : "block");
+	storageListeners.settings.push(function () {
+		globalFeatures.init();
+		featureManager.display(settings.featureDisplay);
+	});
+	storageListeners.userdata.push(function () {
+		globalFeatures.init();
```

Comment:
This reloads more features than it should.

Reply:
It runs the same way as previously.

Reply:
Seems it does, but it should be changed I guess, especially since they are now separated.